RELEASENOTES-1.4.docu

```diff
@@ -376,4 +376,14 @@
       <pre>
       local struct { int value; bool valid; } x = {.valid = false, ...}
       </pre></add-note></build-id>
+  <build-id value="next"><add-note>Template types are now considered
+      serializable <bug number="SIMICS-18507"/>.</add-note></build-id>
+  <build-id value="next"><add-note>Casts of constant object references to
+      template types are now considered constant expressions, allowing for their
+      use as initializers for <tt>session</tt>/<tt>saved</tt> variables:
+      <pre>
+      template t { }
+      group g is t;
+      saved t s = cast(g, t);
```

Comment:
WBN to also allow template references as initializers (as a different PR)

Reply:
Template references? Can you clarify?

Reply:
I meant `saved t s = g;`